src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -62,6 +62,23 @@ def _get_setting_value(self, field_name: str) -> str | None:
             None,
         )
 
+    def _get_sentry_app_installation_uuid(self) -> str | None:
```

Comment:
Getting the Sentry app installation UUID was moved into a separate function to decrease redundant code (used in existing function `self_validate` and new function `render_label`)